backend/codesirius/problems/models/__init__.py

```
@@ -1,5 +1,7 @@
 from problems.models.language import Language
-from problems.models.tag import Tag
 from problems.models.problem import Problem
+from problems.models.reference_solution import ReferenceSolution
+from problems.models.submission import Submission
+from problems.models.tag import Tag
 
-__all__ = ["Language", "Tag", "Problem"]
+__all__ = ["Language", "Tag", "Problem", "ReferenceSolution", "Submission"]
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)